lib/src/callback/callbacks.dart

```diff
@@ -16,7 +16,10 @@ typedef OnWillMoveNext = bool Function(
 typedef SwipableStackItemBuilder = Widget Function(
   BuildContext context,
   int index,
+  int stackIndex,
   BoxConstraints constraints,
+  SwipeDirection direction,
+  double swipeProgress,
```

Comment:
I feel too many parameters could confuse user of this package. So we should make a model which bundle these. 
Could you make a model for this? I'll leave the details of implementation to you, but what I'm imagining is something like the following:
```dart
typedef SwipableStackItemBuilder = Widget Function(
  BuildContext context,
  SwipeProperty property,
);
```